examples/sample-app/README.md

```diff
@@ -104,52 +104,49 @@ the present working directory is the same directory as this README.
+4. Deploy a private Docker registry within OpenShift with the certs necessary for access to master:
 
         $ sudo chmod +r ./openshift.local.certificates/openshift-client/key.key
-        $ pushd ../..
-        $ CERT_DIR=examples/sample-app/openshift.local.certificates/openshift-client hack/install-registry.sh
-        $ popd
+        $ openshift ex registry --create --credentials="${KUBECONFIG}"
+          docker-registry # the service
+          docker-registry # the deployment config
 
-    Note that the private docker registry is using ephemeral storage,
+    Note that the private Docker registry is using ephemeral storage,
     so when it is stopped, the image will be lost. An external volume
     could be used for persistent storage, but that is beyond the scope
     of this tutorial.
 
 5. Confirm the registry is started (this can take a few minutes):
 
-        $ osc get pods
+        $ osc describe service docker-registry
 
     You should see:
 
-        Name                                   Image(s)                    Host                     Labels                                                                                                   Status
-        ----------                             ----------                  ----------               ----------                                                                                               ----------
-        94679170-54dc-11e4-88cc-3c970e3bf0b7   openshift/docker-registry   localhost.localdomain/   deployment=registry-config,name=registrypod,replicationController=946583f6-54dc-11e4-88cc-3c970e3bf0b7   Running
+        Name:       docker-registry
+        Labels:     docker-registry=default
+        Selector:   docker-registry=default
+        Port:       5000
+        Endpoints:  172.17.0.60:5000
+        No events.
 
-6. Confirm the registry service is running.  Note that the actual IP address may vary.
+    If "Endpoints" is listed as <none>, your registry hasn't started yet.  You can run `osc get pods` to
+    see the registry pod and if there are any issues. Once the pod has started, the IP of the pod will
+    be added to the docker-registry service list so that it's reachable from other places.
 
-        $ osc get services
+6. Confirm the registry is accessible (you may need to run this more than once):
 
-    You should see:
-
-        Name                Labels              Selector            IP                  Port
-        ----------          ----------          ----------          ----------          ----------
-        docker-registry                         name=registrypod    172.30.17.3        5001
-
-7. Confirm the registry is accessible (you may need to run this more than once):
-
-        $ curl `osc get services docker-registry -o template --template="{{ .portalIP}}:{{ .port }}"`
+        $ curl `osc get service docker-registry --template="{{ .portalIP}}:{{ .port }}"`
```

Comment:
Proxy won't be exposed to end users in all likelihood.

----- Original Message -----
> >  
> > -    You should see:
> > -
> > -        Name                Labels              Selector            IP
> > Port
> > -        ----------          ----------          ----------
> > ----------          ----------
> > -        docker-registry                         name=registrypod
> > 172.30.17.3        5001
> > -
> > -7. Confirm the registry is accessible (you may need to run this more than
> > once):
> > -
> > -        $ curl `osc get services docker-registry -o template
> > --template="{{ .portalIP}}:{{ .port }}"`
> > +        $ curl `osc get service docker-registry --template="{{
> > .portalIP}}:{{ .port }}"`
> 
> Or you could use the /proxy/services route, right?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1100/files#r25167830